public/js/AdminCtrl.js

```diff
@@ -45,8 +45,8 @@ polr.controller('AdminCtrl', function($scope, $compile) {
                 "ajax": BASE_API_PATH + 'admin/get_admin_links',
 
                 "columns": [
-                    {className: 'wrap-text', data: 'short_url', name: 'short_url'},
-                    {className: 'wrap-text', data: 'long_url', name: 'long_url'},
+                    {data: 'short_url', name: 'short_url'},
+                    {data: 'long_url', name: 'long_url', width: '30%'},
```

Comment:
Ah, I see. Could we remove this change for now? We can handle the width using DataTables' auto width adjustment setting.